pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2264,6 +2264,9 @@ type NetworkSpec struct {
 	// +optional
 	Selectors map[string]string `json:"selectors,omitempty"`
 
+	// +optional
+	NetworkRanges NetworkRangesSpec `json:"networkRanges,omitempty"`
```

Comment:
```suggestion
	NetworkRanges NetworkRangesSpec `json:"networkRanges,omitempty"`
```

I know this is very much self explained, but could you add one liner comment about the field

Reply:
I'm still very much not done with this code, including docs/comments cleanup, and why it's a draft right now. It's probably best to focus only on big-picture feedback right now.